app/javascript/components/users/user/forms/UpdateUserForm.jsx

```diff
@@ -70,7 +70,7 @@ export default function UpdateUserForm({ user }) {
 
   return (
     <Form methods={methods} onSubmit={updateUserAPI.mutate}>
-      <FormControl field={fields.name} type="text" />
```

Comment:
This means it is always readonly - we want to make it readonly only if the user is external